modules/tinymce/src/plugins/media/test/ts/browser/DataAttributeTest.ts

```diff
@@ -10,8 +10,8 @@ describe('browser.tinymce.plugins.media.DataAttributeTest', () => {
   const hook = TinyHooks.bddSetupLight<Editor>({
     plugins: [ 'media' ],
     toolbar: 'media',
-    media_url_resolver: (data: { url: string }, resolve: (response: { html: string }) => void) => {
-      resolve({ html: '<div data-ephox-embed-iri="' + data.url + '" style="max-width: 300px; max-height: 150px"></div>' });
+    media_url_resolver: (data: { url: string }) => {
+      Promise.resolve({ html: '<div data-ephox-embed-iri="' + data.url + '" style="max-width: 300px; max-height: 150px"></div>' });
```

Comment:
I think you are missing returning the promise
```suggestion
      return Promise.resolve({ html: '<div data-ephox-embed-iri="' + data.url + '" style="max-width: 300px; max-height: 150px"></div>' });
```